src/Parser/Core/Modules/MitigationCheck.js

```diff
@@ -50,8 +50,8 @@ class MitigationCheck extends Analyzer{
   on_toPlayer_damage(event){
     const spell = event.ability.guid;
     if(this.checks.includes(spell) && !event.tick){
-      console.log(this.buffCheck);
-      console.log(this.debuffCheck);
+      //console.log(this.buffCheck);
+      //console.log(this.debuffCheck);
       if(this.buffCheck.some((e) => this.selectedCombatant.hasBuff(e))){
```

Comment:
probably just add a ``debug &&`` to these and a debug = false const. So when someone goes looking in this, it'll be easy for them to set debug to true and have it print again